src/main/java/appeng/util/item/AEFluidStack.java

```diff
@@ -144,6 +149,7 @@ public static IAEFluidStack loadFluidStackFromPacket(final ByteBuf data) throws
         fluid.setCountRequestable(countRequestable);
         fluid.setCraftable(isCraftable);
         fluid.setCountRequestableCrafts(countRequestableCrafts);
+        fluid.setUsedPercent(longUsedPercent / 10000f);
```

Comment:
Anything under 0.01% is useless because it is not displayed. (0 is correctly displayed as `<0.01`)
```suggestion
        fluid.setUsedPercent(longUsedPercent / 100f);
```

Reply:
In tooltip displayed percentage up to 4 decimal places

Reply:
> In tooltip displayed percentage up to 4 decimal places

oh ok